app/assets/javascripts/admin/components/site-settings-image-uploader.js.es6

```diff
@@ -0,0 +1,9 @@
+import ImageUploader from "discourse/components/image-uploader";
+
+export default ImageUploader.extend({
+  layoutName: "components/image-uploader",
+
+  uploadUrlParams() {
```

Comment:
If the function doesn't calculate anything, you should be able to do `uploadUrlParams: "..."` - it should still work with inheritance. You can access it with `this.uploadUrlParams`.

Reply:
Do you mean the function is defined like the following?

```
somefunction() {
  // doing something
},

uploadUrlParams: "...",
```

I didn't manage to get the above to work. I tried searching for the "triple dot" notation and most results points to the spread operator.

Reply:
Ah good point :+1: I didn't have to be a function.